lib/matplotlib/axes/_axes.pyi

```diff
@@ -60,13 +60,16 @@ class Axes(_AxesBase):
     @overload
     def legend(self) -> Legend: ...
     @overload
-    def legend(self, handles: Iterable[Artist | tuple[Artist, ...]], labels: Iterable[str], **kwargs) -> Legend: ...
+    def legend(self, handles: Iterable[Artist | tuple[Artist, ...]], labels: Iterable[str],
+               *, loc: LegendLocType | None = ..., **kwargs) -> Legend: ...
```

Comment:
I hope this works. I've pulled `loc` out of kwargs only for the stub; the method signature still does not contain it and handles it via `kwargs` instead. At least mypy doesn't complain, but I don't understand typing well enough whether that's guaranteed to work.

Reply:
It should be fine since we haven't typed `kwargs`, I think.